### ISSUE TITLE
Add Chinese translation support

### ISSUE THREAD

**Issue by PterX** [drakkan/sftpgo#1569]
### Is your feature request related to a problem? Please describe.

First of all, thank you very much for the project you developed, but it is currently inconvenient to use it because it lacks Chinese.

### Describe the solution you'd like

I have completed the translation work on crowdin, please help with review and integration

### What are you using SFTPGo for?

Private user, home usecase (home backup/VPS)

**Comment by drakkan** [comment]
@PterX thank you for your contribution. We are unable to review Chinese ourself, it looks like you have used a translation tool, we also see several spellcheck warnings in Crowdin console

![Schermata del 2024-03-18 14-32-15](https://github.com/drakkan/sftpgo/assets/553263/637da357-f0a6-4bd5-bfdf-cd35e4559fb5)

Have you ever used SFTPGo and tested your translation?

We have provided the translation into Italian and even though it is our native language and we are the authors of the software, I can assure you that it is really challenging to provide a proper translation.

I think we should pay a professional Chinese translator to review your work and ensure that it is of proper quality.

I'm sorry but on this basis your contribution cannot be accepted, however you and other Chinese users can use the translation yourselves and improve it, perhaps it can be merged in the future. Maybe in the meantime we will also have Chinese companies interested in SFTPGo to the point of supporting the project and this will facilitate the review of the translation.

**Comment by PterX** [comment]
Yes, I noticed the warning, this one seems smooth to me, maybe more people's advice is needed

I don't quite understand how to apply translations to my deployed instances yet, if you tell me I'd love to try

Although Chinese is my native language, my mission would be better if I listen to more people's opinions.

**Comment by wakk121** [comment]
While it is easy to deploy a simple file server based on an English interface, the cost of using it for advanced options is high; a multi-language localization would be beneficial for general users as well as attracting more users.

**Comment by wpf973** [comment]
在哪可以下载到您的汉化

**Comment by PterX** [comment]
> 在哪可以下载到您的汉化

https://crowdin.com/project/sftpgo/zh-CN

**Comment by maxbad** [comment]
> > 在哪可以下载到您的汉化
> 
> https://crowdin.com/project/sftpgo/zh-CN

请问这个怎么使用呢, 放到哪个位置啊

**Comment by wpf973** [comment]
> > 在哪可以下载到您的汉化
> 
> https://crowdin.com/project/sftpgo/zh-CN

大佬给个现成的吧

**Comment by drakkan** [comment]
As general rule, please comment in English in this repo. Before distributing a derivative work, please check our [compliance page](https://sftpgo.com/compliance.html). Thank you

**Comment by zhpjy** [comment]
I reviewed this translation and fixed some small problems. It's good enough to use. Though it still has some spellcheck warnings, I think it can be ignored.

**Comment by drakkan** [comment]
We are unable to independently proofread translations into languages ​​other than Italian, and since none of our commercial users seem to be interested, we will not pay a professional to do so.

We may still consider merging translations in 2.7.x on a best effort basis once we implement a way to disable them, which is especially necessary if we decide to accept possible low-quality translations.

Translation support seems to be mostly useful for Chinese users, based on the requests we've received. Actually, if I remember correctly, Chinese users have helped the project with no more than $30 since 2019, if it was only up to them SFTPGo would be an unmaintained project for a few years already

**Comment by leochen12-rgb** [comment]
[translation.json](https://github.com/user-attachments/files/18473977/translation.json)
replace /usr/share/sftpgo/static/locales/it/translation.json

**Comment by drakkan** [comment]
See https://github.com/drakkan/sftpgo/issues/1874

Chinese is low priority for now, sorry

**Comment by CHN-STUDENT** [comment]
> [translation.json 翻译.json](https://github.com/user-attachments/files/18473977/translation.json) replace /usr/share/sftpgo/static/locales/it/translation.json替换 /usr/share/sftpgo/static/locales/it/translation.json

1. Create dir in `C:\ProgramData\SFTPGo\static\locales\cn` or `/usr/share/sftpgo/static/locales/it/translation.json`
2. Download [translation.json 翻译.json](https://github.com/user-attachments/files/18473977/translation.json) 
3. Edit `templates\common\base.html` like this


```
  supportedLngs: ["en","it","cn"],
  fallbackLng: 'cn',
    const lngs = {
        en: { nativeName: 'English' },
        it: { nativeName: 'Italiano' },
        cn: { nativeName: '简体中文' },
    };
```
4. Restart service and enjoy.

By the way, This is a good project. But trying to donate from China is harder than the actual coding..I understand this is a difficult situation. We, as Chinese users, have failed to properly support the creators of this open-source project. We sincerely thank you for your work and apologize for not giving back more to the community.
